src/Button.js

```diff
@@ -39,41 +48,46 @@ DialogButton.defaultProps = {
 
 DialogButton.displayName = "DialogButton";
 
-const styles = StyleSheet.create({
-  button: Platform.select({
-    ios: {
-      flex: 1,
-      justifyContent: "center",
-      alignItems: "center",
-    },
-    android: {
-      justifyContent: "center",
-      alignItems: "center",
-    },
-    web: {
-      justifyContent: "center",
-      alignItems: "center",
-    },
-  }),
-  text: Platform.select({
-    ios: {
-      textAlign: "center",
-      fontSize: 17,
-      backgroundColor: "transparent",
-    },
-    android: {
-      textAlign: "center",
-      backgroundColor: "transparent",
-      padding: 8,
-      fontSize: 14,
-    },
-    web: {
-      textAlign: "center",
-      backgroundColor: "transparent",
-      padding: 8,
-      fontSize: 14,
-    },
-  }),
-});
+const __styles = (isDark) =>
+  StyleSheet.create({
+    button: Platform.select({
+      ios: {
+        flex: 1,
+        justifyContent: "center",
+        alignItems: "center",
+      },
+      android: {
+        justifyContent: "center",
+        alignItems: "center",
+      },
+      web: {
+        justifyContent: "center",
+        alignItems: "center",
+      },
+    }),
+    text: Platform.select({
+      ios: {
+        color: PlatformColor("link"),
```

Comment:
Woah! So long since the last time I used React-Native styles — I didn't know about `PlatformColor`, nice.

Reply:
It appeared in the penultimate version that allows you to recover the native colors of the OS :)

Reply:
Cool cool, this PR is not only adding dark mode then, it's also improving the entire theming UI 🙌